### PR TITLE
Updated Guide to be more up to date

### PR DESCRIPTION
**Pull request by KrauseFx**
Make things easier to understand

### CODE REVIEW

**Comment by KrauseFx** [comment]
Ready to be merged from my side

File: fastlane/docs/Guide.md [hunk 1]
```diff
@@ -1,12 +1,10 @@
-This guide will help you set up Continuous Delivery for your iOS project.
-
-It will help you set up all needed build tools. I tested everything with a fresh Yosemite installation.
+This guide will help you get started with `fastlane` is no time :rocket:
```
Comment (lacostej) [inline fastlane/docs/Guide.md]: is -> in ?

File: fastlane/docs/Guide.md [hunk 2]
```diff
@@ -40,15 +38,15 @@ Install the gem and all its dependencies (this might take a few minutes).
...
+Before changing anything, I recommend committing everything in `git`.
 
 When running the setup commands, please read the instructions shown in the terminal. There is usually a reason they are there.
 
 `fastlane` will create all necessary files and folders for you with the following command. It will use values detected from the project in the working directory.
 
     fastlane init
 
-This will prompt you for your _Apple ID_ in order run [`produce`](https://github.com/fastlane/fastlane/tree/master/produce) which will create your app on iTunes Connect and the Apple Developer Center, and enable [`sigh`](https://github.com/fastlane/fastlane/tree/master/sigh) and [`deliver`](https://github.com/fastlane/fastlane/tree/master/deliver).
+This will prompt you for your _Apple ID_ in order to verify that your application already exists on both iTunes Connect and the Apple Developer Portal. If that's not the case, `fastlane` will ask you if it should be created automatically.
```
Comment (lacostej) [inline fastlane/docs/Guide.md]: init can also initialize for Android / Play Store

File: fastlane/docs/Guide.md [hunk 3]
```diff
@@ -40,15 +38,15 @@ Install the gem and all its dependencies (this might take a few minutes).
 
 ## Setting up `fastlane`
 
-Before changing anything, I recommend committing everything in `git` (in case something goes wrong).
+Before changing anything, I recommend committing everything in `git`.
```
Comment (hemal) [inline fastlane/docs/Guide.md]: Before making any modifications to your project, it's always recommended to commit your current directory in `git`.

File: fastlane/docs/Guide.md [hunk 4]
```diff
@@ -125,16 +118,22 @@ error do |lane, exception|
 end
 ```
 
-You can already try running it, put a line (`say "It works"`) to the `:inhouse` lane and run
+You can already try running `fastlane`, add this to your `Fastfile`
```
Comment (hemal) [inline fastlane/docs/Guide.md]: "You can quickly try running `fastlane` by adding the following to your `Fastfile`:

File: fastlane/docs/Guide.md [hunk 5]
```diff
@@ -125,16 +118,22 @@ error do |lane, exception|
...
+  say "It works"
+end
+```
+
+And then run the following from your Terminal
 
-    fastlane inhouse
+    fastlane example
 
 You should hear your computer speaking to you, which is great!
```
Comment (hemal) [inline fastlane/docs/Guide.md]: If you're successful, you should hear your computer speaking to you!

File: fastlane/docs/Guide.md [hunk 6]
```diff
@@ -125,16 +118,22 @@ error do |lane, exception|
...
 
 A list of available actions can be found in the [Actions documentation](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Actions.md).
 
-Automating the deployment process is a great next step. You should use `increment_build_number` when you want to upload builds to iTunes Connect ([Activate incrementing build numbers](https://developer.apple.com/library/ios/qa/qa1827/_index.html)).
-
+Automating the deployment process is a great next step. You should use the `increment_build_number` action when you want to upload builds to iTunes Connect ([Activate incrementing build numbers](https://developer.apple.com/library/ios/qa/qa1827/_index.html)).
```
Comment (hemal) [inline fastlane/docs/Guide.md]: Do we want to make it app specific and not iOS specific since this works for Android right?

**Comment by KrauseFx** [comment]
Thanks everyone, I'll update the PR tomorrow morning :+1:

**Comment by KrauseFx** [comment]
Ready for another review, feel free to click merge if okay!

**Comment by KrauseFx** [comment]
Merging this now 👍

**Pull request merged**